package.json

```diff
@@ -39,6 +39,7 @@
     "sync-request": "3.0.1",
     "typescript": "1.8.7",
     "validator": "3.33.0",
-    "node-uuid": "1.4.6"
+    "node-uuid": "1.4.6",
+    "nock": "9.0.2"
```

Comment:
I agree.  It seems task specific dependencies should stay in task specific package.json.  Otherwise why do we need it?   In this particular case, nock is a test dependency only required by VCMobileCenterUpload task, and it should stay there.  If dev dependencies are packaged in the vsix, we need to make changes in that process to exclude dev dependencies.